docs/core-concepts/concepts.md

```diff
@@ -10,9 +10,9 @@ Karmada uses the Kubernetes Native API definition for the federated resource tem
 
 ## Propagation Policy
 
-Karmada offers a standalone Propagation(placement) Policy API to define multi-cluster scheduling and spreading requirements.
+Karmada offers a standalone Propagation (placement) Policy API to define multi-cluster scheduling and spreading requirements.
 
-- Support 1:n mapping of policy: workload. Users don't need to indicate scheduling constraints every time creating federated applications.
+- Support 1:n mapping of policy:workload. Users don't need to indicate scheduling constraints every time creating federated applications.
```

Comment:
<ul>
<li> Regarding parentheses and brackets, "a space should be added before the opening parenthesis or bracket, but not before the closing parenthesis or bracket". </li>
<li> No, I found them while reading the docs for my understading. </li>
<li> I hadn't check the ci, it seems that it checks for typos rather than correct punctuation. Also, words such as "Karmada" aren't typical English words, so the Karamda typo would go unchecked. </li>
</ul>